CHANGELOG.md

```diff
@@ -2,26 +2,14 @@
 
 ## Current develop
 
-### Added (new features/APIs/variables/...)
-- Introduced CHANGELOG.md
-
-### Changed (changing behavior/API/variables/...)
```

Comment:
Was the 4 way skeleton removed on purpose or by accident?
I'm in favor of keeping it 4 way.

Reply:
It was an accident. I'll fix.